scripts/reana_bench.py

```diff
@@ -379,42 +365,101 @@ def _save_original_results(workflow: str, df: pd.DataFrame):
     df.to_csv(original_results_path, index=False)
 
 
-@cli.command()
-@click.option("--workflow", "-w", help="Name of the workflow", required=True, type=str)
-@click.option(
-    "--number", "-n", help="Number of workflows to start", required=True, type=int
+def submit(
+    workflow_prefix: str, number_of_workflows: int, file: str, workers: int
+) -> None:
+    """Submit multiple workflows, do not start them."""
```

Comment:
Please add appropriate docs for CLI help, currently the new commands are empty:

```console
$ python ~r/reana/scripts/reana_bench.py --help         
...
Commands:
  analyze  Produce various plots and derive metrics based on launch...
  collect  Collect workflows results, merge them with intermediate...
  launch   Launch benchmark for multiple workflows.
  start
  submit
```

You can also mention there that launch=submit+start so that usage will be clear.

Reply:
Done. I describe `launch` as "Submit and start workflows". This is exactly how it is in the code for the `launch` command.

Reply:
Nice catch with docs. I always forget about them.